README.md

```diff
@@ -1,26 +1,22 @@
 # SuperBenchmark
 
+[![MIT licensed](https://img.shields.io/badge/license-MIT-brightgreen.svg)](LICENSE)
 [![Lint](https://github.com/microsoft/superbenchmark/workflows/Lint/badge.svg)](https://github.com/microsoft/superbenchmark/actions?query=workflow%3ALint)
 [![Codecov](https://codecov.io/gh/microsoft/superbenchmark/branch/main/graph/badge.svg?token=DDiDLW7pSd)](https://codecov.io/gh/microsoft/superbenchmark)
 
-| Azure Pipelines | Build Status |
-| :---: | :---: |
```

Comment:
why move the build status to the end? would be better to keep it at the beginning so that every one could check it at the first glance

Reply:
I think we have a brief build status in the beginning: 
![image](https://user-images.githubusercontent.com/32830091/119287446-17b90c00-bc79-11eb-81d6-a05d50d86b17.png)
Maybe we don't need to have the details at the top of the document.  So I move it to the end for anyone interested in it.

Reply:
OK. Have moved to the beginning.